### DISCUSSION TITLE
Communicated Arduino Mega and JK BMS with RS485 Module ?

### DISCUSSION THREAD

**Discussion by kaanrevan** [syssi/esphome-jk-bms#241] [General]
Hello. We have a project. We need to get the value of the battery we will use in our vehicle from BMS and send it to our Arduino Nextion screen. For this we have the model jk-bd6a24s10 and the RS485 module. However, in our searches, we did not find any wiring example or code sample made with arduino before. I think the connection is easy. RX-TX --> TX-RX and GND... Is this correct? It tells us the total voltage and current of the battery etc. Can you share a few lines that we can use for the code sample showing? How can we do this in Arduino Mega?

**Comment by syssi** [comment]
I cannot provide support how to port the code to another microcontroller. Please make sure to talk to the BMS using a 3.3V TTL. If I remember correctly the logic level of a Arduino Mega is 5V.
